torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -791,7 +767,7 @@ Operator(                                                                      \
         "aten::slice(" decl_type "[] l, int start, int end=9223372036854775807, int step=1) -> " decl_type "[]", \
         listSlice<Shared<c_type>, c_type::ElemType>), \
     Operator( \
-        "aten::append(World w, " decl_type "[] self, " decl_type " el) -> World", \
+        "aten::append(" decl_type "[](a!) self, " decl_type " el) -> " decl_type "[](a!)", \
```

Comment:
This annotation is pretty tricky and is outside what the annotation language can parse. Example:

```
aten::append(Tensor(b)[](a!) self, Tensor(c) el) -> Tensor(b|c)[](a!)
# but the alias set of self's elements _also_ changes after this instruction to include set c as well, we do
# not have a way to denote this.
```

For now we can leave it as it is written, but we need to be careful that list extracts always say the resulting Tensor can be in any alias set until these operators capture this behavior more accurately.